pkg/ipam/crd.go

```diff
@@ -282,41 +282,112 @@ func (n *nodeStore) autoDetectIPv4NativeRoutingCIDR(localNodeStore *node.LocalNo
 	}
 }
 
-// hasMinimumIPsInPool returns true if the required number of IPs is available
-// in the allocation pool. It also returns the number of IPs required and
-// available.
-func (n *nodeStore) hasMinimumIPsInPool(localNodeStore *node.LocalNodeStore) (minimumReached bool, required, numAvailable int) {
+func (n *nodeStore) autoDetectIPv6NativeRoutingCIDR(localNodeStore *node.LocalNodeStore) bool {
```

Comment:
This seems copy-pasty from `autoDetectIPv4NativeRoutingCIDR`, can those be combined?